runtime/crab/src/pallets/staking.rs

```diff
@@ -70,8 +72,20 @@ impl darwinia_staking::Stake for KtonStaking {
 	}
 }
 
-frame_support::parameter_types! {
-	pub const PayoutFraction: sp_runtime::Perbill = sp_runtime::Perbill::from_percent(40);
+pub enum OnCrabSessionEnd {}
+impl darwinia_staking::OnSessionEnd<Runtime> for OnCrabSessionEnd {
+	fn calculate_reward(_maybe_inflation: Option<Balance>) -> Balance {
+		20_000 * UNIT
+	}
```

Comment:
20000 for 1 session( 6 hours) for Crab, so we can calculate the cost for collators staking:

2,920,000 CRAB block rewards for 1 year, much less compared to currently around 32,000,000 CRAB block rewards for 1 year?